Tests/SentryTests/SentrySDKInternalTests.swift

```diff
@@ -778,20 +778,24 @@ class SentrySDKInternalTests: XCTestCase {
 
         let queue1 = createQueue()
         let queue2 = createQueue()
-        let group = DispatchGroup()
 
         let amount = 100
 
-        for i in 0...amount {
+        let expectation = XCTestExpectation(description: "Wait for all measurements to be set")
+        expectation.expectedFulfillmentCount = amount * 2
+        expectation.assertForOverFulfill = true
+
+        for i in 0..<amount {
             setAppStartMeasurement(queue1, i)
             setAppStartMeasurement(queue2, i)
         }
 
         queue1.activate()
         queue2.activate()
-        group.waitWithTimeout(timeout: 100)
 
-        let timestamp = SentryDependencyContainer.sharedInstance().dateProvider.date().addingTimeInterval(TimeInterval(amount))
+        wait(for: [expectation], timeout: 10.0)
+
+        let timestamp = SentryDependencyContainer.sharedInstance().dateProvider.date().addingTimeInterval(TimeInterval(amount - 1 ))
```

Comment:
`l`: Formatting error, shouldn't this be caught by the pre-commit hooks?

```suggestion
        let timestamp = SentryDependencyContainer.sharedInstance().dateProvider.date().addingTimeInterval(TimeInterval(amount - 1))
```